pyqmc/dmc.py

```diff
@@ -92,17 +93,20 @@ def dmc(wf,configs,weights=None, nsteps=1000,tstep=0.02,branchtime=5,accumulator
     df=[]
     for step in range(npropagate):
         if verbose:
-            print("branch step",step)
-        df_,configs,weights = dmc_propagate(wf,configs,weights,
-                tstep,nsteps=branchtime,accumulators=accumulators,
-                stepoffset=branchtime*step+stepoffset,ekey=ekey,
-                verbose=verbose, drift_limiter=drift_limiter)
+            print("branch step",step, flush=True)
+        eloc = accumulators[ekey[0]](configs, wf)[ekey[1]]
+        esigma = np.std(weights*eloc/np.mean(weights))
```

Comment:
You still can't assign this every step; it'll bias you again. Just assign it at the beginning of DMC.